src/test/regress/expected/columnar_indexes.out

```diff
@@ -10,7 +10,7 @@ SET search_path tO columnar_indexes, public;
 --
 create table t(a int, b int) using columnar;
 create index CONCURRENTLY t_idx on t(a, b);
-ERROR:  indexes not supported for columnar tables
+ERROR:  concurrent index commands are not supported for columnar tables
```

Comment:
I think we should early error out for `REINDEX / DROP INDEX CONCURRENTLY` commands in utility hook as we do for `CREATE INDEX CONCURRENTLY` commands (see #4599).

Doing that for `DROP INDEX` commands seems very easy.
However, `REINDEX` commands can also reindex all the indexes in a `SCHEMA` / `DATABASE`.
For such `REINDEX` commands, should we check if there is any columnar table in given `SCHEMA` / `DATABASE` via columnar utility hook ?

I don't see any other solution than this except supporting `CONCURRENT` index commands for columnar tables.

Reply:
Let's work on `CONCURRENTLY` support in another pr